ios/RCTWeChat.m

```diff
@@ -217,13 +274,56 @@ - (void)shareToWeixinWithData:(NSDictionary *)aData scene:(int)aScene callback:(
 {
     NSString *imageUrl = aData[@"thumbImage"];
     if (imageUrl.length && _bridge.imageLoader) {
-        [_bridge.imageLoader loadImageWithTag:imageUrl size:CGSizeMake(100, 100) scale:1 resizeMode:UIViewContentModeScaleToFill progressBlock:nil completionBlock:^(NSError *error, UIImage *image) {
-            [self shareToWeixinWithData:aData thumbImage:image scene:aScene callBack:aCallBack];
-        }];
+        [_bridge.imageLoader loadImageWithTag:imageUrl size:CGSizeMake(100, 100) scale:1 resizeMode:RCTResizeModeStretch progressBlock:nil completionBlock:
+            ^(NSError *error, UIImage *image) {
+                [self shareToWeixinWithData:aData thumbImage:image scene:aScene callBack:aCallBack];
+            }
+         ];
     } else {
         [self shareToWeixinWithData:aData thumbImage:nil scene:aScene callBack:aCallBack];
     }
-    
+
+}
+
+- (void)shareToWeixinWithTextMessage:(int)aScene
+                                Text:(NSString *)text
+                                callBack:(RCTResponseSenderBlock)callback
+{
+    SendMessageToWXReq* req = [SendMessageToWXReq new];
+    req.bText = YES;
+    req.scene = aScene;
+    req.text = text;
+
+    BOOL success = [WXApi sendReq:req];
+    callback(@[success ? [NSNull null] : INVOKE_FAILED]);
+}
+
+- (void)shareToWeixinWithMediaMessage:(int)aScene
+                                Title:(NSString *)title
+                          Description:(NSString *)description
+                               Object:(id)mediaObject
+                           MessageExt:(NSString *)messageExt
+                        MessageAction:(NSString *)action
+                           ThumbImage:(UIImage *)thumbImage
+                             MediaTag:(NSString *)tagName
+                             callBack:(RCTResponseSenderBlock)callback
```

Comment:
Yes, They are just utility functions used internally, by this function:
```
- (void)shareToWeixinWithData:(NSDictionary *)aData
                   thumbImage:(UIImage *)aThumbImage
                        scene:(int)aScene
                     callBack:(RCTResponseSenderBlock)callback
```

They are not export to javascript.